js/app/dashboard/components/context-menus/TasksContextMenu.js

```diff
@@ -53,7 +53,12 @@ const { hideAll } = useContextMenu({
   id: 'task-contextmenu',
 })
 
-async function _assign(tasksToAssign, username, tasksLists, date, dispatch) {
+const useAssignAction = async function(username, tasksToAssign) {
```

Comment:
is this a hook? then it should be a regular function itself and return an async function that will be called from a react component